zipkin-lens/src/components/App/Sidebar.js

```diff
@@ -24,6 +29,24 @@ const pageInfo = {
 };
 
 class Sidebar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.goBackToClassic = this.goBackToClassic.bind(this);
+  }
+
+  goBackToClassic(event) {
+    const { location, history } = this.props;
+
+    Cookies.remove('lens');
+    if (location.pathname === '/zipkin') {
```

Comment:
Zipkin-ui requires trailing slash in root page (`/zipkin`)
so if location is `/zipkin`, add trailing slash